### PR TITLE
feat: Enable logging on auth page via parameter

### PR DESCRIPTION
**Pull request by bennyn**
This PR uses a JS file (`/util/LoggerUtil.js`) inside a TSX file (`/auth/main.tsx`). Because of that we need to tell the `tsconfig.json` to use JS (`"allowJs": true`).

With `"allowJs": true` the TypeScript compiler checks our JavaScript code (which is a good thing!) and fails if it sees type mismatches. I resolved the type mismatches and made the PR compile.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "wire-webapp",
   "license": "GPL-3.0",
   "dependencies": {
+    "@types/js-cookie": "2.2.1",
```
Comment (ffflorian) [inline package.json]: Let's move it to the `devDependencies`.

File: src/script/main/SingleInstanceHandler.js [hunk 2]
```diff
@@ -90,9 +91,8 @@ z.main.SingleInstanceHandler = (() => {
      * Does not check for the id of the running instance and thus cannot be
      * invoked once the registering of the current instance has been done.
      *
-     * @param {Function} listener - A listener to be removed.
      * @throws {Error} - When the current app has already been registered.
-     * @returns {void} - Returns nothing.
+     * @returns {boolean} - Returns nothing.
```
Comment (ffflorian) [inline src/script/main/SingleInstanceHandler.js]: Should return something now :slightly_smiling_face:

File: src/script/util/TimeUtil.js [hunk 3]
```diff
@@ -21,7 +21,7 @@
 
 import moment from 'moment';
 
-import {t} from 'utils/LocalizerUtil';
+import {t} from './LocalizerUtil';
```
Comment (ffflorian) [inline src/script/util/TimeUtil.js:24]: Will we change this everywhere?

File: src/script/main/SingleInstanceHandler.js (hunk 2 above)
Comment (bennyn) [reply src/script/main/SingleInstanceHandler.js]: Why "now"? It has always returned something. Just the JSdoc annotation has been wrong.

File: src/script/util/TimeUtil.js (hunk 3 above)
Comment (bennycode) [reply src/script/util/TimeUtil.js:24]: Talked with @atomrc about it. I have a couple of reasons why we should use relative imports. I will sum them up but for now I would like to change only the ones which prevented `tsc` from compiling.

Comment (atomrc) [reply src/script/util/TimeUtil.js:24]: We could also use absolute path, that would solve the `..` issue and is also compatible with TS. 
https://hackernoon.com/react-pro-tip-use-absolute-imports-for-better-readability-and-easy-refactoring-2ad5c7f2f957
Let's talk about it altogether

**Pull request merged**